source/jormungandr/jormungandr/interfaces/v1/journey_common.py

```diff
@@ -450,6 +450,12 @@ def __init__(self, output_type_serializer):
             "If this list is not empty, we only compute direct_path for modes in this list"
             "And filter all the direct_paths of modes in first_section_mode[]",
         )
+        parser_get.add_argument(
+            "_stop_points_nearby_duration",
+            type=int,
+            hidden=True,
+            help="define the duration between the stop points and the Park&Ride car park",
```

Comment:
Is this only related to Park & Ride or generally used for crow fly ?

```suggestion
            help="define the duration to reach stop points by crow fly",
```